contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -613,6 +627,22 @@ contract OCR2DRRegistry is
     return currentsubscriptionId;
   }
 
+  /**
+   * @notice Gets subscription owner.
+   * @param subscriptionId - ID of the subscription
+   * @return owner - owner of the subscription.
+   */
+  function getSubscriptionOwner(uint64 subscriptionId)
+    external
```

Comment:
Doesn't need to external, right?

Reply:
It does since it is called by OCR2DROracle which is an external contract.